sda-commons-server-s3-testing/README.md

```diff
@@ -2,16 +2,17 @@
 
 [![javadoc](https://javadoc.io/badge2/org.sdase.commons/sda-commons-server-s3-testing/javadoc.svg)](https://javadoc.io/doc/org.sdase.commons/sda-commons-server-s3-testing)
 
-This module provides the [`S3MockRule`](src/main/java/org/sdase/commons/server/s3/testing/S3MockRule.java), 
-a JUnit test rule that is used to automatically bootstrap an AWS S3-compatible object storage instance
+## Extension
+This module provides the [`S3Extension`](src/main/java/org/sdase/commons/server/s3/testing/S3Extension.java),
```

Comment:
For now we should only add JUnit 5 and not remove JUnit 4 documentation / tests / classes.